iaso/models/base.py

```diff
@@ -949,6 +954,14 @@ class Instance(models.Model):
     # See public_create_url workflow in enketo/README.md. used to tell we should export immediately
     to_export = models.BooleanField(default=False)
 
+    class Meta:
+        indexes = [
+            models.Index(fields=["created_at"]),
```

Comment:
I've never tested it myself, but I've read that a  `BrinIndex` is very efficient with large tables (see https://hakibenita.com/9-django-tips-for-working-with-databases#brin-indexes).

Could be worth a try?

```django
from django.contrib.postgres.indexes import BrinIndex
…
BrinIndex(fields=["created_at"]),
```

Reply:
Interesting, I didn't know this! BRIN index indeed sounds perfect for creation timestamps. However in this case, we've (ab)used the `created_at` timestamp until this PR and set it to the creation in of the form in the mobile app. So I think, the "some natural correlation with their physical location within the table" part of why we should use it doesn't apply here.
I think here it'd be safer to go with a regular B-Tree, what do you think?